RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -144,6 +146,13 @@ void AliAnalysisTaskAO2Dconverter::UserCreateOutputObjects()
     Tracks->Branch("fTRDsignal", &fTRDsignal, "fTRDsignal/F");
     Tracks->Branch("fTOFsignal", &fTOFsignal, "fTOFsignal/F");
     Tracks->Branch("fLength", &fLength, "fLength/F");
+    if (fTaskMode) {
+      Tracks->Branch("fPDGcode", &fPDGcode, "fPDGcode/I");
+      Tracks->Branch("fLabel", &fLabel, "fLabel/I");
+      Tracks->Branch("fPDGcodeMother", &fPDGcodeMother, "fPDGcodeMother/I");
```

Comment:
The PDGcodes will be redundant once we store the full kinematics tree separately. Unless there are other vital reason to keep them, I would remove them already at this stage.

Reply:
So far I added the information of TParticle in a separate TTree for Kinematics only.
I have to add now the information on the MC event.
This will be done as for the tracks, attaching more branches to the event tree.